backend/migrating/v2/README.md

```diff
@@ -1,17 +1,113 @@
-# V2 Migrations and Commands
+# V2 Migration Guide
 
 This folder contains scripts and management commands for performing data migrations from v1 to v2 for the Unstract Multitenancy system.
+From version `v0.90.3` onwards, there have been signifcant changes in how data is stored. This warrants a data migration and the below steps can come in handy.
 
-## Overview
+## Preparing for the migration
 
-The commands provided here help you manage the migration process, including creating and dropping schemas and applying the necessary migrations to transition from v1 to v2.
+### Backup existing data
+
+- Make sure `pg_dump` is installed. This would usually be bundled with postgres and can be installed with
+
+```shell
+sudo apt-get install postgresql-client
+```
+
+- Take a backup of the existing data with the below command (provide the appropriate options)
+
+```shell
+pg_dump -h localhost -p 5432 -U unstract_dev -d unstract_db -f unstract-backup-$(date +"%Y-%m-%d_%H-%M-%S").sql
+```
+
+### Create a new database
+
+- Create a new database, for example `unstract_db_v2`
+
+### Dump existing data to this new database
+
+- Exec into the DB if required (from [/docker](/docker/))
+
+```shell
+docker exec -it unstract-db /bin/bash
+```
+
+- Execute the below command to restore the backup in the new DB
 
+```shell
+psql -h localhost -p 5432 -U unstract_dev -d unstract_db_v2 -f unstract-backup-<DateTime>.sql
+```
+
+### Using a feature flag
```

Comment:
We don’t need to mention the feature flag here, as it will be removed in the next few releases. Including it now might cause confusion.